R/rd-r6.R

```diff
@@ -249,38 +250,38 @@ r6_inherited_method_list <- function(block, r6data) {
     "><summary>Inherited methods</summary>"
   )
 
-  c("\\if{html}{",
-    paste0("\\out{", details, "}"),
-    "\\itemize{",
+  c("\\if{html}{\\out{",
+    "<ul>",
```

Comment:
By writing this in HTML, we rely on how the current Rd -> HTML translation works. If they change it from `<ul>` to something else, or they add classes, etc. then our generated docs will be different.